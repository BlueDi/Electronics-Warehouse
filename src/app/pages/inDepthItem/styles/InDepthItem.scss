@import '@common/styles/variables';

h1 {
  margin-left: '5%';
  text-align: 'left';
}
