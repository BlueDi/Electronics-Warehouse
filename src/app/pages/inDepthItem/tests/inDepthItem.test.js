import React from 'react';
import renderer from 'react-test-renderer';
import InDepthItem from '@pages/inDepthItem';

describe('<InDepthItem />', () => {
  test('Renders properly', async () => {
    const component = renderer.create(<InDepthItem id="1" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
