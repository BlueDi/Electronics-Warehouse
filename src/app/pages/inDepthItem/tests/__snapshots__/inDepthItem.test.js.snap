// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InDepthItem /> Renders properly 1`] = `
<div
  className="ui loader"
  text="Preparing Item"
/>
`;
